microbit/src/05-led-roulette/Cargo.toml

```diff
@@ -2,28 +2,29 @@
 name = "led-roulette"
 version = "0.1.0"
 authors = ["Henrik Böving <hargonix@gmail.com>"]
-edition = "2018"
+edition = "2021"
 
 [dependencies.microbit-v2]
```

Comment:
Most of the number corrections (version numbers, runtimes, entry points, sizes, etc) will change quickly as the ecosystem updates. A note to that effect should be put here somewhere, so that we aren't doing this on some regular basis. Probably should just freeze things at some point and make careful notes about the reference point, I guess? I don't know — it's a hard problem.

In a fantasy world, this kind of stuff would be done automatically in CI. But that sounds even harder.